gopro-rename

```diff
@@ -50,31 +51,31 @@ def TryRename(path, dryrun, force, verbose):
         if not dryrun:
             os.rename(path, target_path)
 
-def Process(path, dryrun, force, verbose, recursive):
+def Process(path, dryrun, force, verbose, prefix, recursive):
     if os.path.isfile(path):
-        TryRename(path, dryrun, force, verbose)
+        TryRename(path, dryrun, force, verbose, prefix)
     elif os.path.isdir(path):
         for entry in os.listdir(path):
             if entry == '.' or entry == '..':
                 pass
             elif os.path.isfile(os.path.join(path, entry)):
-                TryRename(os.path.join(path, entry), dryrun, force, verbose)
+                TryRename(os.path.join(path, entry), dryrun, force, verbose, prefix)
             elif recursive and os.path.isdir(os.path.join(path, entry)):
-                Process(os.path.join(path, entry), dryrun, force, verbose, recursive)
+                Process(os.path.join(path, entry), dryrun, force, verbose, prefix, recursive)
 
 def Main():
     parser = argparse.ArgumentParser(description='Goes through files or directories specified on command line and renames files produced by GoPro cameras in a more usable and sortable way')
     parser.add_argument('-n', '--dry-run', action='store_true', help='do not actually rename anything')
     parser.add_argument('-f', '--force', action='store_true', help='rename even if target file exists')
     parser.add_argument('-v', '--verbose', action='store_true', help='print actions to stderr')
     parser.add_argument('-r', '-R', '--recursive', action='store_true', help='process subdirectories recursively')
+    parser.add_argument('-p', '--prefix', nargs='?', default='GoPro', help='optional prefix instead of the default "GoPro"')
```

Comment:
I'd change help string to not duplicate default prefix value. Just `name prefix for renamed files`, or something more nicely worded.

Reply:
Done